Multisig/UI/Settings/OwnerKeyManagement/AddKeyAsNewOwner/AddKeyAsOwnerIntroViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  AddKeyAsNewOwnerViewController.swift
+//  Multisig
+//
+//  Created by Vitaly on 25.04.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class AddKeyAsOwnerIntroViewController: UIViewController, UIAdaptivePresentationControllerDelegate {
+
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet weak var addButton: UIButton!
+    @IBOutlet weak var skipButton: UIButton!
+
+    var onAdd: (() -> ())?
+
+    var onReplace: (() -> ())?
+
+    var onSkip: (() -> ())?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        presentationController?.delegate = self
+
+        // remove underline from navigationItem
+        let navigationBarAppearance = UINavigationBarAppearance()
+        navigationBarAppearance.configureWithTransparentBackground()
+        navigationBarAppearance.backgroundColor = .clear
+        navigationBarAppearance.shadowColor = .clear
+        navigationItem.scrollEdgeAppearance = navigationBarAppearance
+
+        titleLabel.setStyle(.primary)
+        descriptionLabel.setStyle(.secondary)
+        addButton.setText("Add as owner", .filled)
+        skipButton.setText("Skip", .plain)
+    }
+
+    // Called when user swipes down the modal screen
+    func presentationControllerDidDismiss(_ presentationController: UIPresentationController) {
+        onSkip?()
+    }
+
+    @IBAction func didTapAddButton(_ sender: Any) {
+        addOwnerAction()
+
+    }
+
+    @IBAction func didTapSkipButton(_ sender: Any) {
+        onSkip?()
+    }
+
+    func addOwnerAction() {
+
+        let alertController = UIAlertController(
+            title: nil,
+            message: nil,
+            preferredStyle: .actionSheet)
+
+        let add = UIAlertAction(title: "Add new owner", style: .default) { [unowned self] _ in
+            guard alertController.textFields?.first?.text?.trimmingCharacters(in: .whitespacesAndNewlines) == "Add new owner" else {
```

Comment:
can you please explain more why is this guard here? I thought you could just call "onAdd" because you're adding the closure to the 'add' action directly. (same for the "replce")